roster_util.py

```diff
@@ -20,6 +20,8 @@ def add_to_roster(conf, roster, name, username, section, force=False):
         "section": section
     }
 
+    logger.debug("{}".format(roster))
```

Comment:
logger.debug(roster) or is it not always a string?

Reply:
Roster should always be an array.